dotcom-rendering/src/web/components/SecureSignup.tsx

```diff
@@ -54,7 +54,7 @@ const PrivacyTerms = () => {
 
 const RecaptchaTerms = () => (
 	<span css={termsStyle}>
-		We operate Google reCaptcha to protect our website and the Google{' '}
+		We use Google reCaptcha to protect our website and the Google{' '}
```

Comment:
We show this for legal reasons; do we need to confirm what the text should be? I notice it reads as something else on [other pages](https://profile.theguardian.com/signin) on the site

Reply:
Our product manager got an email from data privacy saying we could replace 'operate' with 'use' here (the original question was to the effect of "can we make this text shorter?").  The google's recommended text is here: https://developers.google.com/recaptcha/docs/faq#id-like-to-hide-the-recaptcha-badge.-what-is-allowed

Reply:
Perfect, thanks for the extra detail!